lib/menu.rb

```diff
@@ -0,0 +1,61 @@
+
+require_relative 'italian_menu'
+
+class Menu
+
+SCREEN_WIDTH = 40
+
+attr_reader :selection, :cuisine_choice, :meal, :course, :menu_choice
+# :menu_choice
+# :order
+
+  def initialize(cuisine_choice=ItalianMenu)
+    @cuisine_choice = cuisine_choice
+    @selection = []
+    # @order = order
+    #will inject order_klass later?
+  end
+
+  def menu_choice(meal= :dinner, course= :all)
+    @meal = meal
+    @course = course
+    fail "We only serve lunch and dinner" until meal_choice_valid?
+    fail "Please choose starter, main, dessert or 'all'" until course_choice_valid?
+    @menu_choice = cuisine_choice.new(meal, course)
+  end
+
+
+  def display
+    menu_choice.dishes.each{|key, value| puts("#{key}".ljust(SCREEN_WIDTH) + "#{value}".rjust(SCREEN_WIDTH))}
+  end
+
+
+
+  def select(dish, quantity)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `dish`. If it's necessary, use `_` or `_dish` as an argument name to indicate that it won't be used. You can also write as `select(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Lint/UnusedMethodArgument: Unused method argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. You can also write as `select(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)